ddoc/dub.sdl

```diff
@@ -0,0 +1,4 @@
+name "ddoc_preprocessor"
+description "Preprocessos source code before running Ddoc over it"
+dependency "libdparse" version="0.7.2-alpha.4"
+targetType "executable"
```

Comment:
Do we *have* do use Dub here?

Unless you plan to add dependencies from code.dlang.org, rdmd should suffice.

I don't object to using Dub, just thinking back to all the times it gave us trouble, and don't want to add to that pile.

Reply:
Fair point, but it doesn't change the status quo. The `assert_writeln_magic` script used dub too.

> Unless you plan to add dependencies from code.dlang.org

Yeah, we need libdparse. I'm no huge fan of dub either, either because it seems like we/I are the ones who do all the dog-fooding of it to make it and the registry mature.
Unfortunately, that's also a reason to keep using dub here (instead of doing git clones).